plugins/woocommerce/client/legacy/css/admin.scss

```
@@ -10,6 +10,24 @@
 @import "variables";
 @import "animation";
 @import "fonts";
+@import '@automattic/color-studio/dist/color-variables.scss';
```

```suggestion
@import "@automattic/color-studio/dist/color-variables";
```

to satisfy the WordPress coding standards:

```
13:9   ✖  Expected double quotes                   string-quotes
13:56  ✖  Unexpected extension ".scss" in @import  scss/at-import-partial-extension
```